### PR TITLE
Okazaki - Treatment - P1

### PR DESCRIPTION
**Pull request by wwysoc2**

### CODE REVIEW

**Review by clsackett — APPROVED** [review]

File: src/gdcdictionary/schemas/treatment.yaml [hunk 1]
```diff
@@ -7176,6 +7200,9 @@ properties:
       - Other
       - Unknown
       - Not Reported
+    deprecated_enum:
+      - Brain Stem
+      - Common Duct
```
Comment (ebusygina) [inline src/gdcdictionary/schemas/treatment.yaml]: I can't find the direction to deprecate these enums in [DICT-516](https://gdc-ctds.atlassian.net/browse/DICT-516)

File: src/gdcdictionary/schemas/treatment.yaml [hunk 2]
```diff
@@ -6945,7 +6970,6 @@ properties:
       - Colon
       - Colon - Mucosa Only
       - Common Duct
-      - Conjunctiva
```
Comment (ebusygina) [inline src/gdcdictionary/schemas/treatment.yaml]: I can't find the direction to delete this enum in [DICT-516](https://gdc-ctds.atlassian.net/browse/DICT-516)

**Review by ebusygina — COMMENTED** [review]
I can't find deprecating the "therapeutic_level_achieved" property

File: src/gdcdictionary/schemas/treatment.yaml (hunk 1 above)
Comment (wteouchicago) [reply src/gdcdictionary/schemas/treatment.yaml]: oh weird. I saw in the jira for sample that we are supposed to deprecate these enums, but it's not in the sample PR.

**Comment by wteouchicago** [comment]
were we going to deprecate the `therapeutic_level_achieved` property?

File: src/gdcdictionary/schemas/treatment.yaml (hunk 1 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/treatment.yaml]: Yep, looks like the changes took place for wrong prop. Fixed.

File: src/gdcdictionary/schemas/treatment.yaml (hunk 2 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/treatment.yaml]: Fixed

**Comment by bullenc** [comment]
> I can't find deprecating the "therapeutic_level_achieved" property

@wteouchicago @ebusygina therapeutic_level_achieved" property deprecated 👍

**Review by wteouchicago — APPROVED** [review]

**Review by ebusygina — APPROVED** [review]

**Review by stilesj-uchicago — APPROVED** [review]
DICT-516 looks good to me

**Pull request merged**